client/hbs/editor.hbs

```diff
@@ -34,9 +34,18 @@
 				</button>
 			</footer>
 		</main>
+		<div class='uploader'>
+			<button id='uploadTrigger' class='upload-button upload-item'>Upload New File</button>
```

Comment:
The `id` here doesn't follow the convention used elsewhere.  I recommend switching it to `upload-trigger` for consistency.